kuba-kostrzewski/simpleToDoApp/src/main/java/pl/kostrzej/simpleToDoApp/components/task/TaskConsoleController.java

```diff
@@ -0,0 +1,102 @@
+package pl.kostrzej.simpleToDoApp.components.task;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import pl.kostrzej.simpleToDoApp.components.user.User;
+import pl.kostrzej.simpleToDoApp.components.validator.FieldValidator;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+@Controller
+public class TaskConsoleController implements TaskController {
+
+    private Scanner scanner;
+    private TaskService taskService;
+    private FieldValidator fieldValidator;
+
+    @Autowired
+    public TaskConsoleController(Scanner scanner, TaskService taskService, FieldValidator fieldValidator) {
+        this.scanner = scanner;
+        this.taskService = taskService;
+        this.fieldValidator = fieldValidator;
+    }
+
+    @Override
+    public User addTask(User user){
+        String title, description;
+        Date date;
+        boolean done;
+        do {
+            System.out.println("Podaj tytuł:");
+            title = scanner.nextLine();
+        } while (fieldValidator.isFieldEmpty(title, "Tytuł"));
+        System.out.println("Podaj opis lub zostaw puste:");
+        description = scanner.nextLine();
+        date = readDate();
+        done = readDone();
```

Comment:
Z biznesowego punktu widzenia wydaje mi się, że użytkownik nie powinien mieć wyboru podczas tworzenia czy zadanie jest wykonane czy nie - jakby było wykonane to może nie byłoby sensu go tam umieszczać. Domyślnie niech każde zadanie będzie nie wykonane.